benchmarks/benchmark_filters.py

```diff
@@ -6,12 +6,22 @@
 
 class FiltersSuite:
     """Benchmark for filter routines in scikit-image."""
+    def _filt_func(self, r, c):
+        return np.exp(-np.hypot(r, c) / 1)
+
     def setup(self):
-        self.image = np.random.random((4000, 4000))
-        self.image[:2000, :2000] += 1
-        self.image[3000:, 3000] += 0.5
+        self.image = np.random.random((600, 600))
```

Comment:
why did you decrease the size of the image? I think the idea of having a large enough image was to have an example close to demanding applications, which is more interesting for a benchmark than small images.